source/develop/release-management/features/sla/hosted-engine-network-check.html.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Alternative Network Check for Self Hosted Engine
```

Comment:
Why alternative? DNS should be the default and ICMP a fallback.

Reply:
Done